artio-codecs/src/main/java/uk/co/real_logic/artio/dictionary/generation/CodecConfiguration.java

```diff
@@ -129,6 +131,21 @@ public CodecConfiguration wrapEmptyBuffer(final boolean wrapEmptyBuffer)
         return this;
     }
 
+    /**
+     * Suppresses checks for empty tags, eg |1234=|
+     * instead of rejecting when tag is empty, treat the tag as absent
```

Comment:
Not really, because you're still parsing those fields, the `hasFoo()` methods will return true. I think that's dangerous, because you won't be able to tell a difference between, for example, `|43=N|` and `|43=|`.

Reply:
Oh, damn, good point. I'll revise to make sure we don't parse the tags either and properly treat them as absent.

Reply:
I've now updated the change to not parse the tags, and added tests to assert those fields are absent (rather than present but empty/default).

I've included both String and non-String field tests too (to show it gracefully handles both parsed and non-parsed fields).